reconcile/requests_sender.py

```diff
@@ -49,7 +49,18 @@ def get_encrypted_credentials(credentials_name, user, settings):
 def run(dry_run):
     settings = queries.get_app_interface_settings()
     accounts = queries.get_state_aws_accounts()
-    smtp_client = SmtpClient(settings=settings)
+    smtp_settings = queries.get_smtp_client_settings()
+    smtp_credentials = get_smtp_credentials(
+        secret_reader=SecretReader(settings=settings), secret=smtp_settings.credentials
+    )
+    smtp_client = SmtpClient(
+        host=smtp_credentials.server,
+        port=smtp_credentials.port,
+        username=smtp_credentials.username,
+        password=smtp_credentials.password,
+        mail_address=smtp_settings.mail_address,
+        timeout=smtp_settings.timeout or 30,
```

Comment:
nit: `30` could become `DEFAULT_SMTP_TIMEOUT` since it's been used a couple times?